.eslintignore

```diff
@@ -3,4 +3,5 @@ test/integration/bundle.js
 test/integration/jquery-3.1.0.min.js
 test/integration/helpers.js
 test/integration/lib/first-time.js
-ui/lib/blockies.js
\ No newline at end of file
+ui/lib/blockies.js
+app/scripts/controllers/ts
```

Comment:
We'd ideally add TSLint and use that for all TS files. We're ignore compiled JS files here for the sake of this POC.